Source/DafnyCore/Verifier/Translator.DataTypes.cs

```diff
@@ -673,6 +670,25 @@ partial class Translator {
       }
     }
 
+    /// <summary>
+    /// Return list of variables
+    ///     d: DatatypeValue, T0,T1,...: Ty
```

Comment:
Should `Ty` be `T` here?

Reply:
These comments (through the verifier) are both sketches (that is, they don't highlight every corner case) and (unfortunately) sometimes cryptic. Here, `Ty` is correct, because that's the name of the type we use in Boogie to denote Dafny type terms. The `T` that occurs a few lines down (in `T(T0,T1,...)`) is meant to denote the type to which `T0,T1,...` are type parameters.